doc/man1/openssl-s_client.pod.in

```diff
@@ -589,6 +591,17 @@ For more information on shutting down a connection, see L<SSL_shutdown(3)>.
 There are several known bugs in SSL and TLS implementations. Adding this
 option enables various workarounds.
 
+=item B<-cert_comp>
+
+Enables support for SSL/TLS certificate compression, same as clearing
```

Comment:
Maybe TLS 1.3 instead? (Same comment applies to next section)

Reply:
Fixed